configs/opencv-ovms/scripts/run_grpc_kserv.sh

```diff
@@ -34,7 +34,7 @@ done
 echo "running grpcpython with GRPC_PORT=$GRPC_PORT, DETECTION_MODEL_NAME:$DETECTION_MODEL_NAME"
```

Comment:
Can we rename this script run_grpc_python.sh to now that it support different ways of running the python client.

Reply:
Let's talk about it and do it in a different PR